src/Document/NodeList.php

```diff
@@ -39,6 +40,7 @@ public function __construct(DOMNodeList $list)
      *
      * @return DOMNode
      */
+    #[ReturnTypeWillChange]
```

Comment:
Let's also check warnings in `phpunit.xml` (behaviour changed in a 9.5.x version)